docs/setting_up_your_development_environment.md

```diff
@@ -43,11 +43,17 @@ You need to install:
 1. Run:
    ```terminal
    sudo apt install git make libpq-dev
-   sudo snap install --classic node
    ```
-2. Follow [GitHub CLI's install instructions](https://cli.github.com/)
-3. Follow [Docker's install instructions](https://docs.docker.com/get-docker/) including the [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/). You **don't** need to install Docker Compose
-4. Follow [pyenv's installation instructions](https://github.com/pyenv/pyenv#installation):
+2. Follow [NodeSource's instructions](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions)
+   to install their Node.js Debian/Ubuntu package
+3. Enable Corepack in order to get the `yarn` command
+   (as in [Yarn's install instructions](https://yarnpkg.com/getting-started/install)):
+   ```terminal
+   sudo corepack enable
```

Comment:
I wasn't aware of corepack. Looks like it is a relatively recent addition to Node.